### DISCUSSION TITLE
Terms of use for Renovate's logo

### DISCUSSION THREAD

**Discussion by sisp** [renovatebot/renovate#28778] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

None

### Please tell us more about your question or problem

What are the terms of use for Renovate's logo? I'm asking because I have at least two use cases in mind where it would contribute to a pleasant visual experience, but it seems the legal side of it is unclear:

1. Renovate's logo is included in the Simple Icons collection: https://simpleicons.org/?q=renovatebot Simple Icons is the brand icon library that powers shields.io badge logos, so it may be nice to add a badge to a repository that uses Renovate, e.g.:

     ![Renovate](https://img.shields.io/badge/-Renovate-009485.svg?style=flat-square&logo=renovatebot&labelColor=grey)

1. When Renovate is self-hosted (e.g. on GitLab), a functional/bot user is used for Renovate to run, interact with the platform API and push updates to branches. It would be appealing to use Renovate's logo as the user's avatar, so Renovate's commits and CI pipelines are easily distinguished from those of regular users.

Are these uses of the Renovate logo permitted? Are there any terms of use and/or copyright violations when using the logo in these cases?

### Logs (if relevant)

n/a

**Comment by rarkins** [comment]
I'll check and get back to you. In the past we've been fine with people using this logo for things like their own bot when self-hosting, although it's important that nobody uses the logo and a name which confuses people on a _public_ platform. e.g. calling themselves `@realrenovatebot` on gitlab.com or something like that.

On a related note, we may be retiring/replacing the paintbrush icon for Renovate soon

**Reply by sisp** [reply]
Thanks, @rarkins! If you were to publish formal terms of use somewhere (e.g. the Renovate docs site), then it could also be linked on Simple Icons via the "Brand Guidelines" link on the https://simpleicons.org catalog.

**Reply by HonkingGoose** [reply]
Publishing the brand guidelines in the docs sounds good to me. 🙂

`@rarkins` want me to create a draft guideline for you to review and improve with suggestions?

**Reply by HonkingGoose** [reply, marked as answer]
I'm creating the new Brand guidelines pages with this PR:

- #29021

**Reply by sisp** [reply]
Fatntasic, thank you very much! :pray:

**Reply by HonkingGoose** [reply]
Thank you @sisp for adding the Brand guidelines to the Simple Icons repository:

- https://github.com/simple-icons/simple-icons/pull/10996

I'm updating the brand name use on the Simple Icons repository with:

- https://github.com/simple-icons/simple-icons/pull/11127
